cg/store/crud/read.py

```diff
@@ -1413,14 +1413,17 @@ def get_orders_by_ids(self, order_ids: list[int]) -> list[Order]:
             ids=order_ids,
         ).all()
 
-    def get_order_by_id(self, order_id: int) -> Order | None:
+    def get_order_by_id(self, order_id: int) -> Order:
```

Comment:
Nice, we just need to make sure that if this is used in multiple places, these also handle the error.

Reply:
Yeah 👍 All the places which call this function used to raise this exception, so we are golden